src/models/User.ts

```diff
@@ -0,0 +1,13 @@
+import * as mongoose from 'mongoose';
+
+export interface IUser {
```

Comment:
Yes, we are using `User` [here](https://github.com/Lemoncode/LeanMoodBackend/pull/12/files#diff-e53b954531fcf048d965a0da7fe38af3R6)